controllers/commentsControllers.js

```diff
@@ -31,4 +32,18 @@ function addCommentByArticleId(req, res, next) {
       next(err);
     });
 }
-module.exports = { getCommentsByArticleId, addCommentByArticleId };
+function deleteCommentById(req, res, next) {
+  const { comment_id } = req.params;
+  removeCommentById(comment_id)
+    .then(() => {
+      res.status(204).send({});
```

Comment:
we don't even need the `.send()` here as we're not sending anything we can just have the status :)